config/examples/oauth2.yml

```diff
@@ -1,8 +1,11 @@
-development:
+base: &default
   github:
-    enabled: true
-    client_id: client_id
-    client_secret: client_secret
+    enabled: false
```

Comment:
Why disabling it? this way it'll remain disabled even when the env variables are set correctly